script/openqa-clone-custom-git-refspec

```diff
@@ -7,13 +7,15 @@ Usage:
  openqa-clone-custom-git-refspec <github_branch_url> <openqa_job_url> [CUSTOM_TEST_VAR_1=foo] [CUSTOM_TEST_VAR_2=bar] ...
 
 Options:
- -v, --verbose      execute with verbose output
- -h, -?, --help     display this help
- -n, --dry-run      execute in dry-run mode, do not clone any openQA jobs
+ -v, --verbose          execute with verbose output
+ -h, -?, --help         display this help
+ -n, --dry-run          execute in dry-run mode, do not clone any openQA jobs
+ -c --clone-job-args    pass additional parameters to 'openqa-clone-job', e.g. '--clone-job-args="--show-progress"'
```

Comment:
It may be worth documenting the default parameters as well.

```suggestion
 -c --clone-job-args    pass additional parameters to 'openqa-clone-job', e.g. '--clone-job-args="--show-progress"'
                        the default parameters are '--skip-chained-deps --within-instance'
```

Reply:
good idea! Done.